Jenkinsfiles/Jenkinsfile.cbc-build-app-ami-amzn2

```diff
@@ -15,20 +15,20 @@ pipeline {
   parameters {
     string(
       name: "APP_REPO_BRANCH",
-      defaultValue: "master",
+      defaultValue: "jfuqian/BB2-2336-BB2-Python-upgrade-to-3.8-web-server",
       description: "The branch of the application repo to build in AMI. Defaults to master."
     )
 
     string(
       name: "DEPLOY_REPO_BRANCH",
       // FIXME: Temporary value for testing, revert to master prior to merging!
-      defaultValue: "master",
+      defaultValue: "jfuqian/BB2-2336-BB2-Python-upgrade-to-3.8",
```

Comment:
same here - Should just use the parameter inputs instead of changing the Jenkinsfile